pythermalcomfort/models/phs.py

```diff
@@ -88,19 +91,19 @@ def phs(
     theta : float, optional
         Angle between walking direction and wind direction, [degrees]. Defaults to 0.
     acclimatized : int, optional
-        100 if acclimatized subject, 0 otherwise. Defaults to 100.
+        1 if acclimatized subject, 0 otherwise. Defaults to 1.
     duration : int, optional
         Duration of the work sequence, [minutes]. Defaults to 480.
     f_r : float, optional
-        Emissivity of the reflective clothing, [dimensionless]. Defaults to 0.97.
+        Emissivity of the reflective clothing, [dimensionless]. Defaults to 0.97 in the 2004 standard and 0.42 in the 2023 standard.
     t_sk : float, optional
         Mean skin temperature when worker starts working, [°C]. Defaults to 34.1.
     t_cr : float, optional
         Mean core temperature when worker starts working, [°C]. Defaults to 36.8.
     t_re : float, optional
-        Mean rectal temperature when worker starts working, [°C]. Defaults to False.
+        Mean rectal temperature when worker starts working, [°C]. Defaults to False in the 2004 standard and 36.8 in the 2023 standard.
```

Comment:
this and the following change about t_cr_eq are not 100 correct since even in the old standard this had a default value, since the code was as follows. 

```
    if not t_re:
        t_re = t_cr
    if not t_cr_eq:
        t_cr_eq = t_cr
```

hence we can change the docs to `Mean rectal temperature when worker starts working, [°C]. If False, in the 2004 standard then t_re = t_cr while in the 2023 standard t_re = 36.8 °C`